src/compiler/scala/tools/nsc/backend/jvm/BCodeHelpers.scala

```diff
@@ -710,3 +710,23 @@ trait BCodeHelpers extends BCodeIdiomatic with BytecodeWriters {
 
   } // end of trait JAndroidBuilder
 }
+
+object BCodeHelpers {
+
+  class InvokeStyle(val style: Int) extends AnyVal {
```

Comment:
If you port this, I think it would also make sense to port the rest of 7a12e8110941fb60243453af2d82d0be122ee82d if it's not too hard.

Reply:
I will

Reply:
@smarter It is done